examples/todomvc/index.css

```diff
@@ -14,14 +21,67 @@
 }
 
 
-.CodeMirrorWidget {
-  min-width: 400px;
-}
-
-
 .TodoWidget {
-  overflow: auto;
```

Comment:
I think we still want overflow auto here. Or we wont get scrollbars when the todo list gets long.